documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -13,10 +13,13 @@ The Compute Module has an on-board eMMC device connected to the primary SD card
 
 To flash the Compute Module eMMC, you need the following:
 
-* Another computer, referred to in this guide as the *host device*. You can use Linux (we recommend Raspberry Pi OS or Ubuntu), Windows, or macOS.
+* Another computer, referred to in this guide as the *host device*. You can use Linux (we recommend Raspberry Pi OS or Ubuntu), Windows 11, or macOS.
 * The Compute Module IO Board xref:compute-module.adoc#io-board-compatibility[that corresponds to your Compute Module model].
 * A micro USB cable, or a USB-C cable for Compute Module models since CM5IO.
 
+NOTE: If the Compute Module is not recognised then please remove any USB hubs between the *host device* and the Compute Module.
+For more diagnostic tips see https://github.com/raspberrypi/usbboot?tab=readme-ov-file#troubleshooting
```

Comment:
```suggestion
TIP: In some cases, USB hubs can prevent the host device from recognising the Compute Module. If your host device does not recognise the Compute Module, try connecting the Compute Module directly to the host device. For more diagnostic tips, see https://github.com/raspberrypi/usbboot?tab=readme-ov-file#troubleshooting[the usbboot troubleshooting guide].
```

Suggestion: slight reword to clarify 'remove' terminology; improve formatting of troubleshooting link; ensure that troubleshooting link is part of the admonition